adage/adage/config.py.template

```diff
@@ -131,7 +131,9 @@ AWS_CONFIG = {
         '<configure:http://example.com/tribe_client/get_token>',
     'tribe_scope': 'write',
     'tribe_login_redirect': '/#/home',
-    'tribe_logout_redirect': '/#/home'
+    'tribe_logout_redirect': '/#/home',
+    'public_geneset_folder': os.path.join(OS_CONFIG['home_dir'],
```

Comment:
Aha. Cool, thanks!

So do we need to require a newer version of `tribe-client` now to avoid deployment problems? If so, please update `requirements.txt` to reflect the new minimum version.